godtools/App/Share/SwiftUI Views/SearchBar/SearchBarViewModel.swift

```diff
@@ -11,13 +11,37 @@ import Combine
 
 class SearchBarViewModel: ObservableObject {
     
+    private let getCurrentAppLanguageUseCase: GetCurrentAppLanguageUseCase
+    private let viewSearchBarUseCase: ViewSearchBarUseCase
+    
+    private var cancellables = Set<AnyCancellable>()
+    
+    @Published private var appLanguage: AppLanguageDomainModel = ""
+    
     @Published var cancelText: String = ""
-
-    init(getInterfaceStringInAppLanguageUseCase: GetInterfaceStringInAppLanguageUseCase) {
```

Comment:
Nice I'm glad you were able to update this.  Definitely would like to phase out the GetInterfaceStringInAppLanguageUseCase.